CHANGELOG.md

```diff
@@ -21,6 +21,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   [cyberark/conjur#1987](https://github.com/cyberark/conjur/issues/1987)
 
 ### Fixed
+- Conjur now required the `service-id` param in an authentication request
+  for the OIDC authenticator as there is no use-case without it
```

Comment:
Conjur now requires `service-id` param in an authentication request
  for the OIDC authenticator.

Reply:
@orenbm  Authentication requests for OIDC authenticator now require 'service-id' parameter